src/ape/api/accounts.py

```diff
@@ -41,11 +41,15 @@ def alias(self) -> Optional[str]:
 
     @abstractmethod
     def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
-        ...
+        """
+        Signs message `msg`
```

Comment:
```suggestion
        Signs the given message.
        
        Args:
          msg (:class:`~eth_account.messages.SignableMessage`): The message to sign.
        
        Returns:
          :class:`~ape.types.signatures.MessageSignature` (optional): The signed message.
```

Reply:
I would also add some notes that `SignableMessage` should be handled according to the eth message versions that the signing account implements (e.g. EIP-712, etc.), and that the signature should not be returned if the signer refuses to sign or there is a timeout or other behavior that indicates an error